pxf/pxf-service/src/scripts/pxf-service

```diff
@@ -21,25 +21,25 @@
 #
 
 
-if [ -z $PXF_HOME ]; then
+if [ -z ${PXF_HOME} ]; then
     parent_script_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." && pwd )"
-    env_script=$parent_script_dir/conf/pxf-env.sh
+    env_script=${parent_script_dir}/conf/pxf-env.sh
 else
-    env_script=$PXF_HOME/conf/pxf-env.sh
+    env_script=${PXF_HOME}/conf/pxf-env.sh
 fi
 
 # load pxf-env.sh script
-if [ ! -f $env_script ]; then
+if [ ! -f ${env_script} ]; then
 	echo WARNING: failed to find $env_script
 else
-	source $env_script
+	source ${env_script}
 fi
 
 pxf_user=${PXF_USER}
 instance_port=${PXF_PORT:-51200}
```

Comment:
Just confirming, do we want the service script to default to 51200 if PXF_PORT is not set ?